### ISSUE TITLE
using QML causes crash on 1.9.0-rc2

### ISSUE THREAD

**Issue by brianguenter** [JuliaGraphics/QML.jl#161]
Just installed QML on my machine and tried to use it for the first time. It crashed and killed the REPL.

<details> 
   <summary> Version Information </summary>

```
julia> versioninfo()
Julia Version 1.9.0-rc2
Commit 72aec423c2 (2023-04-01 10:41 UTC)
Platform Info:
  OS: Windows (x86_64-w64-mingw32)
  CPU: 16 × 11th Gen Intel(R) Core(TM) i7-11800H @ 2.30GHz
  WORD_SIZE: 64
  LIBM: libopenlibm
  LLVM: libLLVM-14.0.6 (ORCJIT, tigerlake)
  Threads: 1 on 16 virtual cores
Environment:
  JULIA_EDITOR = code.cmd
  JULIA_STACKTRACE_ABBREVIATED = true
```
</details>

<details> 
   <summary> Error Message </summary>

```
julia> using QML
Assertion failed: jl_datatype_nfields(dt) == 1, file /opt/x86_64-w64-mingw32/x86_64-w64-mingw32/sys-root/usr/local/include/jlcxx/type_conversion.hpp, line 641

[22848] signal (22): SIGABRT
in expression starting at REPL[2]:1
crt_sig_handler at C:/workdir/src\signals-win.c:95
raise at C:\WINDOWS\System32\msvcrt.dll (unknown line)
abort at C:\WINDOWS\System32\msvcrt.dll (unknown line)
assert at C:\WINDOWS\System32\msvcrt.dll (unknown line)
_ZN5jlcxx17boxed_cpp_pointerI10QByteArrayEENS_10BoxedValueIT_EEPS3_P14_jl_datatype_tb at C:\Users\seatt\.julia\artifacts\35e3571ddf1e923f6d00993c84037f8e4843469b\bin\libjlqml.dll (unknown line)
_ZNSt17_Function_handlerIFN5jlcxx10BoxedValueI10QByteArrayEEPKcEZNS0_6Module11constructorIS2_JS5_EEEvP14_jl_datatype_tbEUlS5_E_E9_M_invokeERKSt9_Any_dataOS5_ at C:\Users\seatt\.julia\artifacts\35e3571ddf1e923f6d00993c84037f8e4843469b\bin\libjlqml.dll (unknown line)
_ZN5jlcxx6detail11CallFunctorINS_10BoxedValueI10QByteArrayEEJPKcEE5applyEPKvNS_13WrappedCppPtrE at C:\Users\seatt\.julia\artifacts\35e3571ddf1e923f6d00993c84037f8e4843469b\bin\libjlqml.dll (unknown line)
QByteArray at C:\Users\seatt\.julia\packages\CxxWrap\IdOJa\src\CxxWrap.jl:618
__init__ at C:\Users\seatt\.julia\packages\QML\QJmAc\src\QML.jl:93
jfptr___init___1881 at C:\Users\seatt\.julia\compiled\v1.9\QML\JLkMo_gA4JL.dll (unknown line)
jl_apply at C:/workdir/src\julia.h:1879 [inlined]
jl_module_run_initializer at C:/workdir/src\toplevel.c:75
ijl_init_restored_modules at C:/workdir/src\module.c:982
register_restored_modules at .\loading.jl:1074
_include_from_serialized at .\loading.jl:1020
_require_search_from_serialized at .\loading.jl:1469
_require at .\loading.jl:1750
_require_prelocked at .\loading.jl:1623
unknown function (ip: 0000021afe40277a)
macro expansion at .\loading.jl:1611 [inlined]
macro expansion at .\lock.jl:267 [inlined]
require at .\loading.jl:1574
unknown function (ip: 0000021afe3ff72a)
jl_apply at C:/workdir/src\julia.h:1879 [inlined]
call_require at C:/workdir/src\toplevel.c:466 [inlined]
eval_import_path at C:/workdir/src\toplevel.c:503
jl_toplevel_eval_flex at C:/workdir/src\toplevel.c:731
jl_toplevel_eval_flex at C:/workdir/src\toplevel.c:856
eval_body at C:/workdir/src\interpreter.c:568
jl_interpret_toplevel_thunk at C:/workdir/src\interpreter.c:762
jl_toplevel_eval_flex at C:/workdir/src\toplevel.c:912
jl_toplevel_eval_flex at C:/workdir/src\toplevel.c:856
jl_toplevel_eval_flex at C:/workdir/src\toplevel.c:856
ijl_toplevel_eval at C:/workdir/src\toplevel.c:921 [inlined]
ijl_toplevel_eval_in at C:/workdir/src\toplevel.c:971
eval at .\boot.jl:370 [inlined]
eval_user_input at C:\workdir\usr\share\julia\stdlib\v1.9\REPL\src\REPL.jl:153
repl_backend_loop at C:\workdir\usr\share\julia\stdlib\v1.9\REPL\src\REPL.jl:249
#start_repl_backend#46 at C:\workdir\usr\share\julia\stdlib\v1.9\REPL\src\REPL.jl:234
start_repl_backend at C:\workdir\usr\share\julia\stdlib\v1.9\REPL\src\REPL.jl:231
#run_repl#59 at C:\workdir\usr\share\julia\stdlib\v1.9\REPL\src\REPL.jl:377
run_repl at C:\workdir\usr\share\julia\stdlib\v1.9\REPL\src\REPL.jl:363
jfptr_run_repl_62738.clone_1 at C:\Users\seatt\.julia\juliaup\julia-1.9.0-rc2+0.x64.w64.mingw32\lib\julia\sys.dll (unknown line)
#1019 at .\client.jl:421
jfptr_YY.1019_34713.clone_1 at C:\Users\seatt\.julia\juliaup\julia-1.9.0-rc2+0.x64.w64.mingw32\lib\julia\sys.dll (unknown line)
jl_apply at C:/workdir/src\julia.h:1879 [inlined]
jl_f__call_latest at C:/workdir/src\builtins.c:774
#invokelatest#2 at .\essentials.jl:816 [inlined]
invokelatest at .\essentials.jl:813 [inlined]
run_main_repl at .\client.jl:405
exec_options at .\client.jl:322
_start at .\client.jl:522
jfptr__start_35785.clone_1 at C:\Users\seatt\.julia\juliaup\julia-1.9.0-rc2+0.x64.w64.mingw32\lib\julia\sys.dll (unknown line)
jl_apply at C:/workdir/src\julia.h:1879 [inlined]
true_main at C:/workdir/src\jlapi.c:573
jl_repl_entrypoint at C:/workdir/src\jlapi.c:717
mainCRTStartup at C:/workdir/cli\loader_exe.c:59
BaseThreadInitThunk at C:\WINDOWS\System32\KERNEL32.DLL (unknown line)
RtlUserThreadStart at C:\WINDOWS\SYSTEM32\ntdll.dll (unknown line)
Allocations: 14112010 (Pool: 14095778; Big: 16232); GC: 21
PS C:\tmp\TestQml>
```
</details>

**Comment by rasmushenningsson** [comment]
The crashes seems to have been fixed on the main branch.
Can we expect a new release soon? That would be great. :slightly_smiling_face:

**Comment by ufechner7** [comment]
Perhaps this can be closed now that 0.8 is released?
